packages/fleet/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,13 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -8,8 +8,8 @@
+   catalog.cattle.io/provides-gvr: clusters.fleet.cattle.io/v1alpha1
+   catalog.cattle.io/release-name: fleet
+ apiVersion: v2
+-appVersion: 0.3.4-rc3
++appVersion: 0.3.3
```

Comment:
Should be 0.3.4?

Reply:
Yep. Strange that they were flipped. Fixed now